assets/js/components/Accordion/Accordion.stories.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+import PremiumPill from '@components/PremiumPill';
+import Accordion from '.';
+
+export default {
+  title: 'Accordion',
+  component: Accordion,
+  argTypes: {
+    header: {
+      type: 'string',
+      description:
+        'The content of the accordion header. It can be a plain string or a Component',
+      control: {
+        type: null,
```

Comment:
Thanks for adding this, just a small nitpick. 
Could you change 
`type: null`  to` type: 'string'`
![Screenshot from 2023-04-28 11-35-26](https://user-images.githubusercontent.com/54111255/235112568-baf1a6bc-96b9-4d37-b273-c0a907ff617a.png)

Reply:
as discussed we're keeping the header editable, but not the children at the moment